src/inspector_js_api.cc

```diff
@@ -237,7 +237,6 @@ void IsEnabled(const FunctionCallbackInfo<Value>& args) {
 void Open(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
   Agent* agent = env->inspector_agent();
-  bool wait_for_connect = false;
```

Comment:
Is this argument gone now? Why?

Reply:
Please take a look on changes inside `inspector.js`. `inspector.open` calls new `waitForDebugger` method from JavaScript: https://github.com/nodejs/node/pull/28453/files#diff-cad3afbd7c8a42a25a16c2cfdc7df58aR117

Reply:
Oh. Missed that!